lib/pages/chat/group_chat_empty_view.dart

```diff
@@ -1,67 +1,131 @@
+import 'package:fluffychat/utils/date_time_extension.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/event_extension.dart';
+import 'package:fluffychat/widgets/matrix.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:fluffychat/resource/image_paths.dart';
+import 'package:intl/intl.dart';
 import 'package:linagora_design_flutter/colors/linagora_ref_colors.dart';
+import 'package:matrix/matrix.dart';
 
 class GroupChatEmptyView extends StatelessWidget {
-
-  const GroupChatEmptyView({super.key});
+  final Event firstEvent;
+  const GroupChatEmptyView({super.key, required this.firstEvent});
 
   @override
   Widget build(BuildContext context) {
+    final hasCreatedRoom = firstEvent.type == EventTypes.RoomCreate &&
+      firstEvent.senderId == Matrix.of(context).client.userID;
+
+    return hasCreatedRoom ? _buildOwnGroupChatEmptyView(context) : _buildOthersGroupChatEmptyView(context, firstEvent);
+  }
+
+  Container _buildOthersGroupChatEmptyView(BuildContext context, Event firstEvent) {
     return Container(
-      padding: const EdgeInsets.all(16),
       constraints: const BoxConstraints(
-        maxWidth: 256,
-      ),
-      decoration: BoxDecoration(
-        color: Theme.of(context).colorScheme.onSurfaceVariant.withOpacity(0.08),
-        borderRadius: BorderRadius.circular(16),
+        maxWidth: 224,
       ),
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.center,
-        mainAxisSize: MainAxisSize.min,
         children: [
-          const SizedBox(height: 2,),
-          SvgPicture.asset(ImagePaths.icEmptyGroupChat),
-          const SizedBox(height: 26.0,),
-          Text(
-            L10n.of(context)!.youCreatedGroupChat,
-            style: TextStyle(
-              fontWeight: FontWeight.w700,
-              fontSize: 17,
-              color: Theme.of(context).colorScheme.onSurface,
-            ),
-          ),
+          _getDateWidget(context, firstEvent),
```

Comment:
Why not keep the logic to show the date out of the `empty view`?

Reply:
On the mockup we don't show date when it's our own group 
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/528993bb-909c-4488-94ac-0468fe7f9fc8)

Reply:
these 2 empty groups views wont stay if there is a message or an event, it's a placeholder